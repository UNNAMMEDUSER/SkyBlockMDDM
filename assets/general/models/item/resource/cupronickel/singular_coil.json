{
	"parent": "item/generated",
	"textures": {
		"layer0": "general:item/resource/cupronickel/singular_coil"
	}
}