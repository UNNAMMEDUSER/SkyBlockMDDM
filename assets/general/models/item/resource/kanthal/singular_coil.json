{
	"parent": "item/generated",
	"textures": {
		"layer0": "general:item/resource/kanthal/singular_coil"
	}
}