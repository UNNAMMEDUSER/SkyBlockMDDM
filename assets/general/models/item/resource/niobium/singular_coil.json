{
	"parent": "item/generated",
	"textures": {
		"layer0": "general:item/resource/niobium/singular_coil"
	}
}