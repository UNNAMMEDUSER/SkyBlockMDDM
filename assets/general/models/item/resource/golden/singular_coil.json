{
	"parent": "item/generated",
	"textures": {
		"layer0": "general:item/resource/golden/singular_coil"
	}
}